idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -789,3 +933,21 @@ def default_material_balance_type(blk):
 
     def default_energy_balance_type(blk):
         return EnergyBalanceType.enthalpyTotal
+
+    def calculate_scaling_factors(self):
+        super().calculate_scaling_factors()
+
+        if self.is_property_constructed("sum_component_eqn"):
+            for t, v in self.sum_component_eqn.items():
+                iscale.constraint_scaling_transform(
+                    v,
+                    iscale.get_scaling_factor(self.mole_frac_comp[t]),
+                    overwrite=False)
+
+        if hasattr(self, "visc_d_constraint"):
+            for t, v in self.visc_d_constraint.items():
+                iscale.set_scaling_factor(v, 1e6)
+
+        if hasattr(self, "therm_cond_constraint"):
+            for t, v in self.therm_cond_constraint.items():
+                iscale.set_scaling_factor(v, 1e6)
```

Comment:
Update these in the same way as the scaling for the sum_component_eqn constraint i.e. define default scaling for visc_d and therm_cond like you did for mole_frac_comp, and then use them to set scaling factors for the constraints.

Reply:
@chineduobiora Would you recommend updating the reaction module scaling in this manner as well?

Reply:
Yes, that will be good. However, unlike the physical properties blocks you can't set scaling factors for variables using the set_default_scaling method in the reaction properties block (it doesn't seem to be supported). So instead, I suggest using the set_scaling_factor method to set the scaling factors for variables under the calculate_scaling_factors method, and then you can set the scaling for the related constraints as before. See the example below:

```
    def calculate_scaling_factors(self):
        super().calculate_scaling_factors()

        def _set_default_factor(v, s):
            for i in v:
                if iscale.get_scaling_factor(v[i]) is None:
                    iscale.set_scaling_factor(v[i], s)
        _set_default_factor(self.reaction_rate, 1)
        _set_default_factor(self.rate_constant, 1e2)
```